### PR TITLE
Update chat sample from upstream

### PR DESCRIPTION
**Pull request by prprabhu-ms**
Synced chat sources from upstream at [a541b2](https://github.com/Azure/communication-ui-library/tree/a541b2294943cdd4f885fdfc0ae60511f95c960a/samples/Chat).

### CODE REVIEW

File: Chat/src/app/ChatHeader.tsx [hunk 1]
```diff
@@ -28,6 +30,7 @@ export const ChatHeader = (props: ChatHeaderProps): JSX.Element => {
 
   return (
     <Stack horizontal={true} verticalAlign={'center'} horizontalAlign="end" className={chatHeaderContainerStyle}>
+      <div className={paneButtonContainerStyle}>{}</div>
```
Comment (prprabhu-ms) [inline Chat/src/app/ChatHeader.tsx:33]: Remove this conditional compilation artifact.

File: Chat/src/app/ChatScreen.tsx [hunk 2]
```diff
@@ -82,13 +85,14 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
...
       );
+
     return (
       <Stack className={chatScreenContainerStyle}>
         <Stack.Item className={chatCompositeContainerStyle}>
           <ChatComposite
             adapter={adapter}
             fluentTheme={currentTheme.theme}
-            options={{ topic: false }}
+            options={{ autoFocus: 'sendBoxTextField', participantPane: !hideParticipants }}
```
Comment (prprabhu-ms) [inline Chat/src/app/ChatScreen.tsx:95]: Upstream note: `participantPane` should only be set for beta builds.

File: Chat/src/app/theming/SwitchableFluentThemeProvider.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT license.
 
 import React, { useState, useMemo, createContext, useContext } from 'react';
-import { FluentThemeProvider, lightTheme, darkTheme } from '@azure/communication-react';
+import { FluentThemeProvider, lightTheme, darkTheme } from '@internal/react-components';
```
Comment (prprabhu-ms) [inline Chat/src/app/theming/SwitchableFluentThemeProvider.tsx:5]: Replace `@internal` references with `@azure/communication-react`

Upstream note.: Just like storybook, we should force the use of external name in samples.

File: Chat/src/app/ChatHeader.tsx [hunk 4]
```diff
@@ -3,6 +3,7 @@
 
 import React from 'react';
 import { DefaultButton, Icon, IconButton, mergeStyles, Stack } from '@fluentui/react';
+import { People20Filled, People20Regular } from '@fluentui/react-icons';
```
Comment (prprabhu-ms) [inline Chat/src/app/ChatHeader.tsx:6]: upstream note: These should be imported only on beta build.

File: Chat/src/app/ChatHeader.tsx [hunk 5]
```diff
@@ -11,6 +12,7 @@ import {
   leaveButtonStyle,
   leaveIcon,
   leaveIconStyle,
+  paneButtonContainerStyle,
```
Comment (prprabhu-ms) [inline Chat/src/app/ChatHeader.tsx:15]: Upstream note: `setHidParticipants` should only be added for beta builds

File: Chat/src/app/ChatHeader.tsx (hunk 1 above)
Comment (prprabhu-ms) [reply Chat/src/app/ChatHeader.tsx:33]: Done.

File: Chat/src/app/ChatScreen.tsx (hunk 2 above)
Comment (prprabhu-ms) [reply Chat/src/app/ChatScreen.tsx:95]: Done.

File: Chat/src/app/theming/SwitchableFluentThemeProvider.tsx (hunk 3 above)
Comment (prprabhu-ms) [reply Chat/src/app/theming/SwitchableFluentThemeProvider.tsx:5]: Done

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**